supported-cf-cli-commands.md

```diff
@@ -244,3 +248,17 @@ This section describes the metadata operations that Application Service Adapter
 | `cf labels` | Y | The `app`, `org`, and `space` resources are supported. |
 | `cf set-label` | Y | The `app`, `org`, and `space` resources are supported. |
 | `cf unset-label` | Y | The `app`, `org`, and `space` resources are supported. |
+
+## <a id="role-management"></a> User Role Management
+
+This section describes the operations for adding and removing roles that Application Service Adapter supports.
```

Comment:
Are we currently supporting any roles other than "SpaceDeveloper"? If not, it seems a bit odd to claim that we support `set-org-role`